cmd/crashd.go

```diff
@@ -4,27 +4,32 @@
 package cmd
 
 import (
+	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/vmware-tanzu/crash-diagnostics/logging"
 
 	"github.com/vmware-tanzu/crash-diagnostics/buildinfo"
 )
 
 const (
-	defaultLogLevel = logrus.InfoLevel
-	CliName         = "crashd"
+	defaultLogLevel    = logrus.InfoLevel
+	CliName            = "crashd"
+	defaultLogFileName = "crashd.log"
```

Comment:
I think we probably need to modify this and instead of using a default name we need to add a uuid onto it. Otherwise if a user runs two different scripts concurrently the logs will fight over the file and causing data to be lost.

Reply:
I think if user is allowed to override log file name at CLI, then I woudn't worry about name clashing.